app/views/works/show.html.erb

```diff
@@ -20,7 +20,7 @@
   <% component.with_row(label: 'DOI', values: [@work_presenter.doi_value]) %>
   <% component.with_row(label: 'Collection', values: [@work_presenter.collection_link]) %>
   <% component.with_row(label: 'Depositor', values: [@work_presenter.depositor]) %>
-  <% component.with_row(label: 'Version details', values: [@work_presenter.version]) %>
+  <% component.with_row(label: 'Version', values: [@work_presenter.user_version]) %>
```

Comment:
change label in form and show the user_version instead of the system version from cocina